rollup.config.mjs

```diff
@@ -22,16 +21,15 @@ export default [
       }
     ],
     plugins: [
-      coffee(),
-      nodeResolve({ extensions: [".coffee"] }),
+      nodeResolve({ extensions: [".js"] }),
```

Comment:
Not a blocker, but I think we can just call `nodeResolve()` now here and in the next instance below, since `.js` should be the default